samples/drivers/CAN/README.rst

```diff
@@ -19,25 +18,22 @@ In loopback mode, the board receives its own messages. This could be used for
 standalone testing.
 
 The sample can be built and executed on boards supporting CAN.
-The output ports and pins of the LEDs can be configured by Kconfig.
+The LED output pin is defined in boards device tree.
+
+For the reel_board CAN expansion board, use the provided shield.
```

Comment:
Can you provide a link reference to the "CAN expansion board" and "the provided shield".  I'm not sure what these are.  Who or what is providing the shield?

Reply:
Thanks for the review! The "CAN expansion board" is not yet available publicly.
I'll add the comment and links again when it's available.